esp32/modules/services.py

```diff
@@ -1,59 +1,250 @@
-import uos
-
 # File: services.py
+# Version: 2
+# API version: 1
 # Description: Background services for SHA2017 badge
 # License: MIT
 # Authors: Renze Nicolai <renze@rnplus.nl>
 #          Thomas Roos   <?>
 
-def setup():
+import uos, ujson, easywifi, easyrtc, time, appglue, deepsleep, ugfx, badge, machine
+
+services = [] #List containing all the service objects
+loopCallbacks = {} #Dict containing: {<FUNCTION>:<Wifi required on next run>}
+drawCallbacks = [] #List containing draw functions
+
+def setup(pmCb=False, drawCb=False):
     global services
+    global loopCallbacks
+    global drawCallbacks
+    
+    if pmCb:
+        print("[SERVICES] Power management callback registered")
+        global pmCallback
```

Comment:
I encountered it where it's used below, and it took me way too long to find this place where it's indeed clearly stated. Anyway, no big deal, but these kinds of improvements really help a lot.